libs/bluegl/bluegl-gen.py

```diff
@@ -235,6 +237,17 @@ def generateProxies(api, functions, output_dir, platforms):
 %(function)s:
     mov __blue_gl%(suffix)s_%(function)s@GOTPCREL(%%rip), %%r11
     jmp *(%%r11)
+''',
+        'LinuxAArch64': '''
+	.align	2
+	.global	%(function)s
+	.type	%(function)s, %%function
+%(function)s:
+	adrp	x0, :got:__blue_gl%(suffix)s_%(function)s
+	ldr	x0, [x0, #:got_lo12:__blue_gl%(suffix)s_%(function)s]
+	ldr	x0, [x0]
+	br	x0
+	.size	%(function)s, .-%(function)s
```

Comment:
I don't think you can clobber `x0` like that, can you?

Reply:
It looks like you can use `x16` which is what the Android code is using in its `libGLESv3.so`.

Reply:
Sure, switched to `x16`.